packages/app-store/exchangecalendar/pages/setup/index.tsx

```diff
@@ -119,7 +120,7 @@ export default function ExchangeSetup() {
                         <SelectField
                           label={t("exchange_authentication")}
                           options={authenticationMethods}
-                          defaultValue={authenticationMethods[0]}
+                          defaultValue={authenticationMethods[1]}
```

Comment:
nit: can we store this in a variable with a self-explanatory name

Reply:
Not really, but will update with a better version.